### DISCUSSION TITLE
Backtest parameters

### DISCUSSION THREAD

**Discussion by richwomanbtc** [nkaz001/hftbacktest#90] [Q&A] [answered]
I would like to know the parameters used in the backtest for reproducibility and follow-up testing, are they listed anywhere? Or are all the strategy parameters the same (0.0005) except for symbol-specific parameters such as tick_size?

**Comment by richwomanbtc** [comment]
Also, although this may be difficult, would it be possible to provide the data used for backtesting?

**Reply by Billy1900** [reply]
tick-level data is not free. so we might not be able to provide it.

**Reply by richwomanbtc** [reply]
If so, can I use the historical tick-level data here?
https://www.binance.com/en/landing/data
I can access this.

**Reply by Billy1900** [reply]
if you have the access, definitely, you can use it.

**Reply by nkaz001** [reply]
the last time i checked binance's official data, its quality was insufficient. i recommend using tardis.dev data instead to get the exact same result as the example, although it also has its flaws in exchange timestamp. collecting data yourself might be the best approach, but you cannot have the historical data.

**Reply by richwomanbtc** [reply]
Thank you.
I am not currently subscribed to tardis data and will consider it again. Personally, it would be useful to be able to handle binance data directly, so I will consider how to parse that.
Regarding the binance data, as far as I can see at hand, it seems to behave correctly if you replay the order book taking into account the update_id. Of course, I can't reproduce the latency.

**Reply by nkaz001** [reply]
if you collect data yourself, please see [collect](https://github.com/nkaz001/collect-binancefutures) and [Data Preparation](https://hftbacktest.readthedocs.io/en/latest/tutorials/Data%20Preparation.html#For-HftBacktest-in-Python,-use-the-Python-version-of-the-Data-Collector).

**Comment by Billy1900** [comment]
> I would like to know the parameters used in the backtest for reproducibility and follow-up testing, are they listed anywhere? Or are all the strategy parameters the same (0.0005) except for symbol-specific parameters such as tick_size?

https://github.com/nkaz001/hftbacktest/blob/876d1e3ada7fb16240fae23a2e4fa3118f6368a7/hftbacktest/__init__.py#L177C1-L192C3. Please refer to here.

**Reply by richwomanbtc** [reply]
Sorry, I may have asked a slightly confusing question. My aim is to reproduce the backtest here.
https://github.com/nkaz001/hftbacktest/blob/master/rust/examples/gridtrading_256assets.ipynb
However, this does not appear to describe the detailed conditions of the backtest. The intention of my question was to find out about this detailed setup.

**Reply by nkaz001** [reply, marked as answer]
it's the same as [gridtrading.ipynb](https://github.com/nkaz001/hftbacktest/blob/master/rust/examples/gridtrading.ipynb) but it used more assets in [tickers_256.json](https://github.com/nkaz001/hftbacktest/blob/master/rust/examples/tickers_256.json). so you're correct. the parameters are the same except for symbol-specific parameters. may the better to find the more optimal parameters using [gridsearch](https://github.com/nkaz001/hftbacktest/blob/master/rust/examples/6_gridsearch.py).
